receiver/hostmetricsreceiver/hostmetrics_receiver_test.go

```diff
@@ -74,6 +74,13 @@ var resourceMetrics = []string{
 	"process.disk.io",
 }
 
+var semConvMetrics = map[string]string{
```

Comment:
Do we need this map? It seems like there is never going to be a situation when it is desirable to use different schema urls for different metrics, even in the future. So why not just mandate that all host metrics should use the same schema url as defined in `conventions.SchemaURL`?

Reply:
I thought we agreed on setting `SchemaURL` only for the scrapers/metrics that use semantic convention attributes. Initially I added it to all the scrapers, now it's added only for scrapers that use the semantic convention attributes. The host metrics receiver has many scrapers that export different metrics. I can set the schema for all hostmetrics scrapers even if they don't use semantic convention attributes. I'm good with that.

Reply:
I probably misunderstood what this map is. So this lists the metric names that are used in any of the scrapers and are expected to have the SchemaURL set?
Please add a comment to explain.

Reply:
> I probably misunderstood what this map is. So this lists the metric names that are used in any of the scrapers and are expected to have the SchemaURL set?

Yes, this is a list of metrics with semantic conventions attributes